package.json

```diff
@@ -1,10 +1,28 @@
 {
+  "devDependencies": {
+    "stylelint": "^13.13.1",
+    "stylelint-config-standard": "^21.0.0",
+    "stylelint-csstree-validator": "^1.9.0",
+    "stylelint-scss": "^3.21.0",
+    "webpack-dev-server": "^3"
+  },
   "dependencies": {
-    "@rails/webpacker": "5.4.4",
+    "@babel/preset-react": "^7.16.7",
+    "@rails/webpacker": "5.4.3",
+    "axios": "^0.25.0",
+    "babel-plugin-transform-react-remove-prop-types": "^0.4.24",
+    "babel-polyfill": "^6.26.0",
+    "prop-types": "^15.8.1",
+    "react": "^18.1.0",
+    "react_ujs": "^2.6.2",
+    "react-dom": "^18.1.0",
+    "react-redux": "^8.0.1",
+    "react-router-dom": "^6.3.0",
+    "redux": "^4.2.0",
```

Comment:
- [ ] the required dependency "babel-loader" is not installed please make sure to do so.